### PR TITLE
Add required package name for vmstat

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by Maxusmusti — DISMISSED** [review]

**Review by dbutenhof — APPROVED** [review]
Well, I guess if it's needed, it's needed.

**Review by npalaska — DISMISSED** [review]

File: agent/tool-scripts/base-tool [hunk 1]
```diff
@@ -186,6 +186,9 @@ virsh-migrate)
 	longoptions="${longoptions},vm:"
 	tool_package_name="libvirt-client"
 	;;
+vmstat)
+	tool_package_name="procps-ng"
```
Comment (ndokos) [inline agent/tool-scripts/base-tool:190]: procps-ng needs to be deleted from the spec file.

**Review by ndokos — CHANGES_REQUESTED** [review]

File: agent/tool-scripts/base-tool (hunk 1 above)
Comment (portante) [reply agent/tool-scripts/base-tool:190]: Fixed.

File: agent/containers/images/Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ _REPO_TEMPLATE = ../../ansible/pbench/agent/roles/pbench_repo_install/templates/
 
 # The list of RPMs which provide the various tools we offer.
 # Not intended to be overridden with an environment variable.
-_TOOL_RPMS = pbench-sysstat perf blktrace bpftrace numactl pcp-system-tools strace kernel-tools tcpdump
+_TOOL_RPMS = atomic-openshift-clients blktrace bpftrace cpupowerutils golang kernel-tools libvirt-client nmap-ncat numactl pbench-sysstat pcp-system-tools perf procps-ng strace tcpdump trace-cmd
```
Comment (dbutenhof) [inline agent/containers/images/Makefile:32]: So what happened here?

Comment (portante) [reply agent/containers/images/Makefile:32]: The long of the short of it is that the list was incomplete, and when @ndokos pointed out that we needed to remove `procps-ng` from the agent RPM `.spec` file, I checked the `agent/tool-scripts/base-tool` file to see what packages were recorded there for tools.  I just took that list, bumped it against the existing, and presented them in sorted order.

**Review by dbutenhof — DISMISSED** [review]

**Review by npalaska — DISMISSED** [review]

**Review by ndokos — DISMISSED** [review]

**Review by dbutenhof — APPROVED** [review]

**Review by ndokos — APPROVED** [review]

**Pull request merged**